src/web100srv.c

```diff
@@ -307,9 +307,6 @@ void cleanup(int signo) {
 
   sigsafe_debug_log(1, signo, "Received a signal");
   switch (signo) {
-    default:
```

Comment:
We should keep the `default:` case. That way if rogue signals happen we will get log messages about them.